Makefile

```diff
@@ -187,8 +187,7 @@ install-python-tests:
 
 ## python-unittests: Run pythoon unit tests
 python-unittests:
-	DEBUG=true VERBOSE=true pytest -s \
-	          packages/cli packages/sdk
+	DEBUG=true VERBOSE=true pytest --cov --cov-branch
```

Comment:
You removed `-s` key and list of tests to run and now running `pytest` without test suites specification, is it fine?

Reply:
I added back -s key.
It's from pytest --help. And as I understand **-s** would show only sterr and not stout.
 --capture=method      per-test capturing method: one of fd|sys|no|tee-sys.
 -s                    shortcut for --capture=no.

and directories for testing (in case if we will not use test coverage) but they are also indicated in .coveragerc